### ISSUE TITLE
Make compatible with the FreshAnimations resource pack [REQUEST]

### ISSUE THREAD

**Issue by TiKhangTran** [haubna/PhysicsMod#368]
When the mobs ragdoll they fracture with the limbs still linked with the main body. In simpler terms, the mob would ragdoll, but fall apart simultaneously.

**Comment by caoqt1** [comment]
Fresh animations alter the mob models which will break physics, sorry.

**Comment by DonFlymoor** [comment]
Perhaps a copy of the default mob could replace the modified one upon death?

**Comment by haubna** [comment]
Closed due to moving feature suggestions to the [discord](https://discord.gg/VsNs9xP) (channel #feature-suggestions)
